index.html

```diff
@@ -1,123 +1,231 @@
-                                <a class="menu__link" href="#">Аксессуары</a>
-                            </li>
-                            <li class="menu__item">
-                                <a class="menu__link" href="#">Офисные принадлежности</a>
-                            </li>
-                        </ul>
-                    </nav>
-                    <div class="header__fill"></div>
-                    <div class="header__button">
-                        <button class="header-button-icon header-button-icon_search">
-                            Search
-                        </button>
+
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>HTML и CSS</title>
+    <link rel="stylesheet" href="css/styles.less" />
+</head>
+
+<body>
+    <header class="header">
+        <div class="column-wrapper">
+            <div class="header__content">
+                <div class="header__logo">
+                    <a href="/">
+                        <img class="logo" src="css/blocks/logo/logo.svg" alt="hh.ru" />
+                    </a>
+                </div>
+                <nav class="header__menu">
+                    <ul class="menu">
+                        <li class="menu__item">
+                            <a class="menu__link" href="#">Одежда</a>
+                        </li>
+                        <li class="menu__item">
+                            <a class="menu__link" href="#">Сумки</a>
+                        </li>
+                        <li class="menu__item">
+                            <a class="menu__link" href="#">Аксессуары</a>
+                        </li>
+                        <li class="menu__item">
+                            <a class="menu__link" href="#">Офисные принадлежности</a>
+                        </li>
+                    </ul>
+                </nav>
+                <div class="header__fill"></div>
+                <div class="header__button">
+                    <button class="header-button-icon header-button-icon_search">
+                        Search
+                    </button>
+                </div>
+                <div class="header__button">
+                    <button class="header-button-icon header-button-icon_login">
+                        Войти
+                    </button>
+                    <button class="header-button header-button_login">Войти</button>
+                </div>
+                <div class="header__button">
+                    <button class="header-button-icon header-button-icon_menu">
+                        Menu
+                    </button>
+                </div>
+            </div>
+        </div>
+    </header>
+
+    <section class="section-slider">
+        <div class="column-wrapper">
+            <div class="column-row">
+                <div class="column column_s-2 column_m-3 column_l-6 order_s-3">
```

Comment:
модификатор должен всегда относиться к какому-то блоку или элементу
В данном случае, судя по названию, `order_s-3` относиться к блоку order, которого вообще нет.
Можно вот так сделать
```suggestion
                <div class="column column_s-2 column_m-3 column_l-6 column_order-s-3">
```

Reply:
поправил